app/src/main/java/edu/uc/groupProject/topten/ui/main/MainViewModel.kt

```diff
@@ -21,12 +21,17 @@ class MainViewModel : ViewModel() {
     var listService : ListService = ListService()
     var firestoreService : FirestoreService =  FirestoreService()
     var playAnimation:Boolean = true
+    var lists : MutableLiveData<ArrayList<String>> = MutableLiveData<ArrayList<String>>()
```

Comment:
As the ViewModels, in this case, should only be one, I decided to use the MainViewModel. Thus, I added the few missing parts from PastListsViewModel to the MainViewModel (I did not feel comfortable to delete the other 2 ViewModels, so I just left them as is).